README.md

```diff
@@ -6,38 +6,26 @@ This library can receive and parse data from JIRA webhook, convert it into easil
 It comes loaded with a few [default listeners](#default-listeners) for the most common scenarios, but you can [extend it](#customizing) with your own custom logic too.
 
 
-# Installation
-To install this library clone the code from GitHub and run `composer install` in the root folder.
+# Installation and configuration
+To install this library simply pull it in through composer
+```        
+    "require": {
+        "kommuna/vicky": "dev-master",
+    },
+```   
+and run `composer install`.
 
 #### 1. Config files  
 Copy `bot/config.example.php` to `/etc/SlackBot/config.php` and `client/config.example.php` to `/etc/vicky/config.php` and follow the instructions in them to set your values.
 
-Get your bot token [here](https://my.slack.com/services/new/bot).
+You can get your slack bot token [here](https://my.slack.com/services/new/bot).
 
-Configure Slack bot, for details see [this] (https://github.com/jclg/php-slack-bot).
+>IMPORTANT: Please make sure that you have the webserver and the slackbot running on different ports. The slackbot port is configurable and you can set it in the `/etc/slackBot/config.php` file. 
 
-#### 2. Server/Virtual host
```

Comment:
I actually removed this section completely, cause now people are using it as a package and the server is not really Vicky's "concern" anymore. They would have to have a server set up for their app anyway, maybe even just as a route from a bigger app.